lib/heroku_pg_backups_archive/toolbelt_helper.rb

```diff
@@ -17,6 +17,11 @@ def fetch_backup_info(backup_id)
 
       private
 
+      def follower_db(app)
+        output = HerokuHelper.run_heroku_command("pg:info --app #{app} | grep Followers | head -n 1")
+        return output.blank? ? "" : output.split(" ").last
+      end
+
```

Comment:
@niallsmart @BrentWheeldon 

- I considered that but it felt like it was just something that would break.  For instance, I just upgraded our follower DB from premium-0 to premium-2, which required provisioning a new instance.  That would mean I'd have to remember to update this ENV variable.  I don't see much point in having that level of control.  I'm honestly surprised capturing from a follower isn't the default in Heroku.
- Yes. It just takes the last listed follower, but they should all be essentially equivalent.  Some may be a few commits behind others but that amounts to mere seconds of discrepancy.
- Commit with tests coming right now